### PR TITLE
Seyit file uploaded

### PR DESCRIPTION
**Pull request by SeyitAliOran**
Seyit file uploaded

### CODE REVIEW

File: src/ISIMLER/SeyitAli.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ISIMLER;
+
+public class SeyitAli {
+    public static void main(String[] args) {
+        System.out.println("Seyit Ali local");
+    }
```
Comment (bilaloender) [inline src/ISIMLER/SeyitAli.java:7]: abc dogrulamasi uygulanmali

Comment (SeyitAliOran) [reply src/ISIMLER/SeyitAli.java:7]: @bilaloender Hata duzeltildi [240fe3b](https://github.com/Ozleblebici/B11_Git_Practice_Project/pull/2/commits/240fe3bcfec0e53f682ff633226e0f0a5baeb676)

Comment (bilaloender) [reply src/ISIMLER/SeyitAli.java:7]: tesekkur ederim

**Review by bilaloender — APPROVED** [review]
your class approved

**Pull request merged**